ws-nextjs-app/makefile

```diff
@@ -0,0 +1,15 @@
+BUILD_DIR=~/Documents/workspace/simorgh
```

Comment:
This is my checkout location for simorgh, this will need adjusting for your location